### ISSUE TITLE
Error: broken pipe while commiting data to server (development environment)

### ISSUE THREAD

**Issue by mschnitzer** [openSUSE/open-build-service#1977]
I set up a development environment and after I created my home project, I tried to checkout it locally (what did work). But when I try to commit my changes to the API server, I get the following error message:
```
manuel@manuel:~/osc/home:Admin/pawncc> osc commit
WARNING: SSL certificate checks disabled. Connection is insecure!

Sending    pawncc-3.10.20150531.tar.gz
Sending    pawncc.changes
Sending    pawncc.spec
Transmitting file data .Failed to reach a server:
 [Errno 32] Broken pipe
```

I also tried to run `osc -A http://127.0.0.1:3000 commit` but it did not work.

I found the following error string in my backend logs:
```
[bfaf89cf-0bd1-430f-93c4-f054b04bd91c] [8163:1428.83] ForbidsAnonymousUsers::AnonymousUser Anonymous user is not allowed here - please login /vagrant/src/api/app/mixins/forbids_anonymous_users.rb:8:in `be_not_nobody!'
```

More logs: https://gist.github.com/mschnitzer/f9d001ee33b5c056a4c05dc5c39d6fcc

**Comment by lethliel** [comment]
Looks more like a performance issue of the box. 
Should be resolved with new development boxes. 
The error message in the backend logs has nothing to do with the broken pipe.

**Comment by lethliel** [comment]
fixed with #2045
